src/marks/raster.js

```diff
@@ -393,8 +394,8 @@ function blend(a, ca, b, cb, c, cc) {
 }
 
 function pick(random = Math.random) {
-  return (a, ca, b, cb, c) => {
-    const u = random();
+  return (a, ca, b, cb, c, cc, x, y, z) => {
+    const u = random(x, y, z);
```

Comment:
`z` is always zero here. The concept of a step count makes sense for the random-walk interpolation, but maybe for barycentric interpolation we should only pass the x and y.